mantle/kola/harness.go

```diff
@@ -272,6 +273,73 @@ func testRequiresInternet(test *register.Test) bool {
 	return false
 }
 
+type DenyListObj struct {
+	Pattern string   `yaml:"pattern"`
+	Tracker string   `yaml:"tracker"`
+	Streams []string `yaml:"streams"`
+	Arches  []string `yaml:"arches"`
+}
+
+type ManifestData struct {
+	AddCommitMetadata struct {
+		FcosStream string `yaml:"fedora-coreos.stream"`
+	} `yaml:"add-commit-metadata"`
+}
+
+func parseDenyListYaml() {
+	var objs []DenyListObj
+
+	// Parse kola-denylist into structs
+	denyListFile, err := ioutil.ReadFile("src/config/kola-denylist.yaml")
+
+	if err != nil {
+		return
+	}
+
+	plog.Debug("Found kola-denylist.yaml")
```

Comment:
Maybe just a little more info. Something like this maybe:

```suggestion
	plog.Debug("Found kola-denylist.yaml. Processing listed denials.")
```